packages/react-portal-compat/package.json

```diff
@@ -27,21 +27,16 @@
   },
   "devDependencies": {
     "@fluentui/eslint-plugin": "*",
-    "@fluentui/react-conformance": "*",
-    "@fluentui/react-conformance-griffel": "9.0.0-beta.3",
+    "@fluentui/react-shared-contexts": "9.0.0-rc.4",
     "@fluentui/scripts": "^1.0.0"
   },
   "dependencies": {
-    "@fluentui/react-theme": "9.0.0-rc.4",
-    "@fluentui/react-utilities": "9.0.0-rc.5",
-    "@griffel/react": "1.0.0",
     "tslib": "^2.1.0"
   },
   "peerDependencies": {
+    "@fluentui/react-shared-contexts": "^9.0.0-rc.1",
```

Comment:
I decide to go with `peerDependencies` there as people will have `@fluentui/react-components` in their apps.

Other option (_probably more logical_) that I see is to a peer dependency on `@fluentui/react-components` instead.

Reply:
I would say let's use `@fluentui/react-components` instead. Since our current recommendation is to still use that package when using fluent

Reply:
@ling1726 done in 0300999.

Reply:
^ I like that